addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -15,6 +17,7 @@ var _t = core._t;
 
 const OdooEditor = OdooEditorLib.OdooEditor;
 const isBlock = OdooEditorLib.isBlock;
+const rgbToHex = OdooEditorLib.rgbToHex;
```

Comment:
Note : colorpicker.js also has a function for this so actually I had a choice... Future refactor could involve moving stuff from one side to the other to avoid duplicate logic.